internal/bulk.go

```diff
@@ -42,8 +42,12 @@ type BulkResponse struct {
 	IsFinal bool `json:"is_final"`
 }
 
+type BulkOptions struct {
```

Comment:
Given this is a public struct (capital B at the beginning) it should have a comment.

Reply:
Thanks! Will add it